stack/production.yaml

```diff
@@ -0,0 +1,389 @@
+acute-care:
```

Comment:
Should we delete all the individual stacks at the same time? We'll have to be careful not to revert any changes to the individual stacks accidentally, so probably best to regenerate this file right before merging (at the same time as deleting all the individual stacks).